src/vulkan/vulkan_renderer.h

```diff
@@ -11,27 +11,28 @@
 namespace IC {
     class VulkanRenderer : public Renderer {
     public:
-        VulkanRenderer(RendererConfig &config);
-        ~VulkanRenderer();
+        VulkanRenderer(const RendererConfig &config);
+        virtual ~VulkanRenderer();
 
-        void AddMesh(Mesh &meshData, Material &materialData);
-        void DrawFrame();
+        void AddMesh(Mesh &meshData, Material &materialData) override;
```

Comment:
does override do anything or just make it more explicit that this function is overriding?

Reply:
It's mostly just for being explicit and isn't required. What is _does_ do is throw a compiler error if the base class ever removes the function, so it's just nice to have really.